fe/fe-core/src/main/java/com/starrocks/planner/DeltaLakeScanNode.java

```diff
@@ -167,7 +168,8 @@ private void addPartitionLocations(Map<PartitionKey, Long> partitionKeys, Partit
         if (!partitionKeys.containsKey(partitionKey)) {
             partitionId = nextPartitionId();
             String tableLocation = deltaLakeTable.getTableLocation();
-            Path filePath = new Path(tableLocation, file.getPath());
+            Path p = new Path(URI.create(file.getPath()));
```

Comment:
Could you add some comments in the code to explain why calling `URI.create` is necessary?